src/main/resources/META-INF/rewrite/jaxb-plugins.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2024 the original author or authors.
+# <p>
+# Licensed under the Moderne Source Available License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://docs.moderne.io/licensing/moderne-source-available-license
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.javax.MigrateJaxBWSPlugin
+displayName: Migrate JAXB-WS Plugin
+description: Upgrade the JAXB-WS Maven plugin to be compatible with Java 11.
+tags:
+  - java11
+recipeList:
+  - org.openrewrite.maven.ChangePluginGroupIdAndArtifactId:
+      oldGroupId: org.jvnet.jax-ws-commons
+      oldArtifactId: jaxws-maven-plugin
+      newGroupId: com.sun.xml.ws
+      newVersion: 2.3.5
```

Comment:
Why 2.3.5 specifically and not say 2.3.7, or 2.3.+?
https://mvnrepository.com/artifact/com.sun.xml.ws/jaxws-maven-plugin

Reply:
Fixed in https://github.com/openrewrite/rewrite/commit/d330b7eac63d206d1ca2b5e0d643ad42e63df83a ; can we adopt `2.3.7` here as suggested above?